test/ng/documentSpec.js

```diff
@@ -27,3 +27,29 @@ describe('$document', function() {
     });
   });
 });
+
+
+describe('$$isDocumentHidden', function() {
```

Comment:
I think there should be a test tgat shows the value changing when `visibilitychange` is fired.